src/components/app.jsx

```diff
@@ -1,17 +1,9 @@
-import React, { Fragment } from 'react';
-import ReactCreateElementComponent from 'Task1/reactCreateElement';
-import ReactComponent from 'Task1/reactComponent';
-import ReactPureComponent from 'Task1/reactPureComponent';
-import ReactFunctionalComponent from 'Task1/reactFunctionalComponent';
+import React from 'react';
+import SearchPage from 'Components/pages/searchPage/searchPage';
+import 'Styles/main.less';
 
 const App = () => (
-  <Fragment>
-    <div>Blank message</div>
-    <ReactCreateElementComponent />
-    <ReactComponent />
-    <ReactPureComponent />
-    <ReactFunctionalComponent />
-  </Fragment>
+  <SearchPage />
```

Comment:
I see some of components & containers are wrapped by the error boundary, but you can wrap the whole app tree with it, right? Let's keep it simple.

Reply:
@loginov-rocks Yes, it was implement with idea that in case of error in some of simpler component, it will not crash the whole app

Reply:
@loginov-rocks fixed